src/pyrenew/metaclass.py

```diff
@@ -126,6 +126,38 @@ def _assert_sample_and_rtype(
     return None
 
 
+def compute_incidence_observed_with_delay(
```

Comment:
`observed` isn't quite the right word. The result of this function is often something that is still `latent`.

Reply:
Will use `compute_delay_ascertained_incidence` for now, we can rename if we come up with something better